docs/connectors/gitter.md

```diff
@@ -0,0 +1,22 @@
+# opsdroid connector Gittter
+
+A connector for [Gitter](https://developer.gitter.im/docs/welcome).
+
+## Requirements
+
+To use the Gitter connector you will need a user for the bot to use and generate a personal access-token. It is recommended that you create a separate user from your own account for this. 
+
+### Creating your application
+
+- Create Gitter user for the bot to use and log into it
+- Create a [access-token](https://developer.gitter.im/apps)
+- Get the [room-id](https://developer.gitter.im/docs/rooms-resource) of room you want to listen. 
```

Comment:
Could we allow the room name here also? Making the user call an API to get the room ID is a bit of a complex step.

Reply:
For that we have to parse all the list to get userId.We can raise separate pr for same if it's fine.

Reply:
Fine with me!